proc_CPMG.py

```diff
@@ -5,31 +5,31 @@
 fl = figlist_var()
 logger = init_logging('info')
 for searchstr, exp_type, nodename, postproc, label_str in [
-        #('200221','CPMG_TEMPOLgel_3p0_1','deadtime=5'),
+        #('200221_CPMG_TEMPOLgel_3p0_1','test_equip','signal','spincore_CPMG_v1','deadtime=5'),
         ('200221_CPMG_TEMPOLgel_2p9_1','test_equip','signal','spincore_CPMG_v1','deadtime=5'),
-        #('200304','CPMG_2p6_1','deadtime=5'),
-        #('200305','CPMG_3p5_2','deadtime=5'),
-        #('200305','CPMG_3p6_2','deadtime=5'),
-        #('200305','CPMG_3p7_2','deadtime=5'),
-        #('200305','CPMG_3p7_3','deadtime=5'),
-        #('200305','CPMG_3p8_2','deadtime=5'),
-        #('200305','CPMG_3p9_2','deadtime=5'),
-        #('200305','CPMG_4p0_1','deadtime=5'),
```

Comment:
the commented lines were reverted to an earlier version -- is there a reason why?

Reply:
probably the mistake I was making earlier of instead of just checking out a previous version I converted and forgot to convert back to the newest edition